app/models/pokemon.js

```diff
@@ -14,7 +14,7 @@ var pokemonSchema = new Schema({
     userId     : {type: String},
     deviceId   : {type: String},
     category   : {type: [Boolean]},
-    gender     : {type: String, enum: ['m', 'f', 'u', 'h']},
+    gender     : {abbreviation: {type: String, enum: ['m', 'f', 'u', 'h']}, male_ratio: Number, female_ratio: Number},
```

Comment:
:D Happy to see that you correct the code even before I come in with comments :D